gazelle/python/BUILD.bazel

```diff
@@ -29,22 +29,20 @@ go_library(
     importpath = "github.com/bazel-contrib/rules_python/gazelle/python",
     visibility = ["//visibility:public"],
     deps = [
-        "//manifest",
         "//pythonconfig",
         "@bazel_gazelle//config:go_default_library",
         "@bazel_gazelle//label:go_default_library",
         "@bazel_gazelle//language:go_default_library",
-        "@bazel_gazelle//language/proto:go_default_library",
```

Comment:
This was added in an early version of #3057 and then code was changed to no longer need it.

:+1: to remove.